rental.rb

```diff
@@ -0,0 +1,12 @@
+class Rental
+  attr_accessor :date
+  attr_reader :person, :book
```

Comment:
- **person**, **book** are both Getter and Setter. you should also declare them with `attr_accessor`(Which is also a shortcut method) instead of `attr_reader`
[Please Open here to learn more ](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_principles_examples.md#useful-shortcuts-attr_reader-attr_writer-and-attr_accessor)